### PR TITLE
fix: do not show inherit toggle if items are empty

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA

Must upgrade mirinae version after merge [this PR](https://github.com/cloudforet-io/mirinae/pull/239)

### CODE REVIEW

File: src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetMoreOptions.vue [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+});
+const selectedOptions = ref<MenuItem[]>([]);
+const {
+    visibleMenu: visibleContextMenu,
+    refinedMenu,
+    contextMenuStyle,
+    showContextMenu,
+    hideContextMenu,
+    initiateMenu,
+} = useContextMenuController({
```
Comment (sulmoJ) [inline src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetMoreOptions.vue:83]: Praise : whoa...! I forgot about this composable.

File: src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 2]
```diff
@@ -124,7 +124,17 @@ export default defineComponent<Props>({
...
             widgetConfig: computed(() => (props.widgetConfigId ? getWidgetConfig(props.widgetConfigId) : undefined)),
             widgetOptionsJsonSchema: {} as JsonSchema,
-            inheritableProperties: computed<string[]>(() => state.widgetConfig?.options_schema?.inheritable_properties || []),
+            inheritSchemaProperties: computed(() => {
+                const variablesSchema = dashboardDetailState.variablesSchema;
+                const properties = state.widgetConfig?.options_schema?.schema?.properties ?? {};
+                const schema = {};
+                Object.keys(properties).forEach((propertyName) => {
+                    const propertySchema = properties[propertyName] ?? {};
+                    if (!propertySchema) return;
```
Comment (sulmoJ) [inline src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:133]: Question: Isn't `propertySchema` always `true`?

**Review by sulmoJ — APPROVED** [review]
LGTM!!
Master of Refactoring..!

File: src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue (hunk 2 above)
Comment (WANZARGEN) [reply src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:133]: Oh yes. I just added if statement without thinking 😅 thank u~

Comment (WANZARGEN) [reply src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue:133]: I will fix it after PR~

**Pull request merged**